### ISSUE TITLE
change password

### ISSUE THREAD

**Issue by danielstock** [eclipse/mosquitto#913]
i changed the user password by command mosquitto_passwd.
should i restart the mosquitto.exe to validate it?

**Comment by ralight** [comment]
At the moment the easiest answer is "yes" There is a mechanism to trigger a reload of the password file, but it's difficult to access on Windows. I'm going to leave this issue open to remind me to take a look when I'm back in Windows.

**Comment by ralight** [comment]
Yes, this is the same for all of the files that mosquitto reads. On Unix like systems you can send a SIGHUP signal to carry out a reload, but that method isn't possible on Windows.

**Comment by electrocoder** [comment]
What should I do to change user password at runtime? Any progress?

**Comment by ptone** [comment]
Related: https://github.com/eclipse/mosquitto/issues/524

I don't think PW, at least remote_password gets reloaded on SIGHUP
